src/api/addressBook.ts

```diff
@@ -0,0 +1,7 @@
+export const addressBook = {
```

Comment:
Ah, never mind. I thought this directory contains functions that will make API calls. Now I see how it's being used.